src/apps/competitions/tests/test_submissions.py

```diff
@@ -72,27 +69,17 @@ def test_max_submissions_not_counting_other_user_submissions(self):
     def test_submission_not_created_if_max_reached(self):
         self.set_max_submissions(per_person=1)
         self.make_submission()
-        try:
-            self.make_submission(name='Find Me')
-        except PermissionError:
-            try:
-                Submission.objects.get(name='Find Me')
-                assert False, "Submission should not have been created"
-            except Submission.DoesNotExist:
-                pass
+        self.assertRaises(PermissionError, self.make_submission)
+        assert not Submission.objects.filter(name='Find Me').exists()
 
+    def test_children_submissions_dont_count_toward_max(self):
+        self.make_submission(parent=self.make_submission())  # Should only count as one submission
```

Comment:
in general, imo, comments should be above the code and not to the right, but it definitely fits in some cases -- seems a bit much here